README.es-ES.md

```diff
@@ -0,0 +1,567 @@
+
+Para reducir el tiempo de renderizado, prueba lo siguiente:
+- Aplicar scroll virtual a través de [CDK](https://material.angular.io/cdk/scrolling/overview) o [ngx-virtual-scroller](https://github.com/rintoj/ngx-virtual-scroller)
+- Reducir la cantidad de elementos DOM representados en la sección `* ngFor` de su plantilla. Por lo general, los elementos DOM innecesarios / no utilizados surgen al ampliar la plantilla una y otra vez. Volviendo a pensar su estructura probablemente hará las cosas más fáciles.
+- Utilizar [`ng-container`](https://angular.io/guide/structural-directives#ngcontainer) cuando sea posible.
+
+**Recursos**
+
+- ["directiva NgFor"](https://angular.io/docs/ts/latest/api/common/index/NgFor-directive.html) - official documentation for `*ngFor`
+- ["Mejorar rendimiento de Angular utilizando trackBy"](https://netbasal.com/angular-2-improve-performance-with-trackby-cc147b5104e5) - muestra algunos GIFs con esta práctica.
+- [Component Dev Kit (CDK) Virtual Scrolling](https://material.angular.io/cdk/scrolling/overview) - API description
+- [ngx-virtual-scroller](https://github.com/rintoj/ngx-virtual-scroller) - Muestra una lista "virtual" infinita.
+
+### Optimizar expresiones en plantilla (Template expressions)
+
+Angular ejecuta expresiones de plantilla después de cada ciclo de detección de cambios. Los ciclos de detección de cambios se activan mediante muchas actividades asíncronas, como resoluciones de promesa (promise), resultados http, eventos de temporizador (timer), pulsaciones de teclas y movimientos del ratón.
+
+Las expresiones deben terminar rápidamente o la experiencia de usuario puede empeorar, especialmente en dispositivos más lentos. Considere cachear los valores cuando su cálculo es costoso.
+
+**Recursos**
+- [quick-execution](https://angular.io/guide/template-syntax#quick-execution) - Documentación oficial para expresiones en plantilla (template expressions)
+- [Increasing Performance - more than a pipe dream](https://youtu.be/I6ZvpdRM1eQ) - ng-conf video in youtube. Using pipe instead of function in interpolation expression
+- [Increasing Performance - more than a pipe dream](https://youtu.be/I6ZvpdRM1eQ) - Vídeo en youtube de ng-conf. Usando Pipe en vez de funciones de interpolación.
+
+# Conclusión
+
+La lista de prácticas evolucionará dinámicamente a lo largo del tiempo con prácticas nuevas / actualizadas. En caso de que note que falta algo o si cree que se puede mejorar cualquiera de las prácticas, no dude en reportar un problema y/o un PR. Para más información, por favor, eche un vistazo a la sección inferior "[Contribuyendo](#contributing)".
+
+# Contribuyendo
+
+En caso de que note que falta algo, incompleto o incorrecto, una "pull request" será muy apreciada. For discussion of practices which are not included in the document please [open an issue](https://github.com/mgechev/angular2-performance-checklist/issues).
```

Comment:
Here seems you need to remove the original English and the paragraph is duplicated. 

> En caso de que note que algo falta, está incompleto o es incorrecto, una _pull request_ será muy apreciada. Para tratar sobre prácticas no incluidas, por favor, [abra un _issue_](https://github.com/mgechev/angular2-performance-checklist/issues).

Good job! Interesting documentation 😄